pkg/realserver/configure.go

```diff
@@ -338,6 +379,96 @@ func (r *realserver) periodic() error {
 	}
 }
 
+// ConfigureHAProxy this function is the bridge between a v6 address and a v4
+// pod address. This function iterates over the declared v6 configs and backends
+// for each, checks if any pods match that service selector, and creates a
+// haproxy instance for each backend that maps the VIP:PORT to a list of backend
+// these are the pod ips, not the service IPs, to ensure traffic stays on-node
```

Comment:
we should expand on this comment. what is the output of the function?  how many files is it generating? why are the generated files the way that they are?